### ISSUE TITLE
Validator doesn't accept WMS GetCapabilities URL and returns error: Returned HTTP status code was '403' (Forbidden )

### ISSUE THREAD

**Issue by marianhaluska** [INSPIRE-MIF/helpdesk-validator#878]
Hello,

I'am triyng to validate one of our inspire datasets, but the validator seems to have an issue with my service URL. Upon staring the test I get an error "An error has occured.". I checked the console in my browser and found this error: "Failed to load resource: the server responded with a status of 400 ()" (I'am also attaching the responseJSON from the web browser). It seems that the server is not accepting outside requests.

The tested url for WMS: https://geo.shmu.sk/geoserver/am-aqmz/ows?service=wms&version=1.3.0&request=GetCapabilities 

So far I was not able to pinpoint the cause of the issue. I suspect that the problem is somewhere on main server where our GeoServer is installed.

Can somesone help me with this issue.
Thank you very much in advance.

Marian
[Browser responceJSON.txt](https://github.com/INSPIRE-MIF/helpdesk-validator/files/10054645/Browser.responceJSON.txt)

PS: Does INSPIRE Validator have any restrictions regarding the version of GeoServer that you can use for production environment? In other words, can newer vesion of GeoServer impact INSPIRE Validator. In this case we upgraded from GeoServer 2.16.1 to 2.21.2.

**Comment by jenriquesoriano** [comment]
Dear @marianhaluska , 

thanks for your question.
According to your logs, it seems that you had implemented some security grants in your services, so the connection from the INSPIRE Reference Validator to your service was rejected by your server. Please see message: 
"The server requires authentication. If authentication was provided, then the authenticated user is not permitted to perform the requested operation. If the problem persists, please contact the administrator of the server geo.shmu.sk."

On the other hand, we have tested today your service and it was able to connect, so maybe you already removed those access security constraints. So the issue would be solved. 
Please see the produced [test report](https://inspire.ec.europa.eu/validator/test-reports/details.html?id=EID10baec51-b72c-4a65-9380-21e1c5d448aa).

Please let us know if it is all correct from your side.

Best regards,

**Comment by marianhaluska** [comment]
Dear @jenriquesoriano 

Our IT support contacted me this morning, stating that the problem was on our side, namely no the side of our aplication firewall. The firewall stopped the request. Problem was fixed in the morning, I was able to start the validation in the morning hours. Now (at 2:41 PM) the Validator seems to by down because of maintenance. So I will resume testing as soon as the Validator is back online. 

And yes, the problem seems to be solved, so this issue can be closed.

Best regards,
